core/cmd/local_client_ocr.go

```diff
@@ -72,10 +77,24 @@ func (cli *Client) listOCRKeyBundles(c *clipkg.Context) error {
 		`***********************************************************************************
 Encrypted Off-Chain Reporting Key Bundles in DB
 ***********************************************************************************`)
-	for keyidx, key := range keys {
-		fmt.Println("ID                ", key.ID)
-		fmt.Println("On-chain Address  ", "0x"+hex.EncodeToString(key.OnChainSigningAddress[:]))
-		fmt.Println("Off-chain PubKey  ", hex.EncodeToString(key.OffChainPublicKey))
+	for keyidx, enckey := range keys {
+		password, err := getPassword(c)
+		if err != nil {
+			return err
```

Comment:
this will error our the command if any of the keys don't match the password. Maybe should just skip them instead?